gramhopper/configuration/triggers_reponses_parsers.py

```diff
@@ -22,7 +22,9 @@ def parse_single(cls, config, global_elements):  # pylint: disable=unused-argume
         mapping_cls = cls.mapping_class()
         element_cls = mapping_cls[config_copy.pop('type')]
 
-        return element_cls(**config_copy)
+        if isclass(element_cls):
```

Comment:
Some triggers are classes (most of them actually), but some are instances (mostly filter triggers). This allows both cases to be used.

Reply:
Hmm.. Then the name "element_cls" isn't so good.
Additionally, why do we have this ambiguity?

Reply:
1. You are right. I'll rename.
2. Filter triggers are mostly instances of `FilterBasedTrigger` receiving `BaseFilter`. I could have created some lambda to make it feel like a class constructor, but it would make filter_trigger.py look like shit.